tox.ini

```diff
@@ -72,6 +72,6 @@ convention = google
 
 [gh-actions]
 python =
-  3.6: py36
-  3.7: py37, check-dist, lint, docs, coverage
-  3.8: py38
+  3.7: py37, check-dist, lint
+  3.8: docs, coverage
+  3.9: py39
```

Comment:
we're starting to add 3.10, as well - it has turned up a couple interesting breakages; up to you.

Reply:
As for NeuroR, I tried locally and the build failed at compiling scipy because I am missing BLAS/LAPACK libraries. I guess it's a bit early to officially support it.

Reply:
Weird; they seem to have wheels:
https://pypi.org/project/scipy/#files

Is scipy pined somewhere?

Reply:
After our discussion with Adrien yesterday, the issue was due to the pandas constraint in tox. However, python3.10 seems to be out of the game until `Neuron` releases wheels for it.